### ISSUE TITLE
[Bug] Already processed files are being reprocessed

### ISSUE THREAD

**Issue by dslowder** [Josh5/unmanic#29]
After picking up the commits to master from April, I'm now seeing already converted files being reprocessed: https://www.dropbox.com/s/vl9mm3bq3rn29rk/Screenshot%202019-04-24%2012.29.22.png?dl=0

```root@24df4303db8b:/$ ffprobe /library/Season\ 01/Breaking\ Bad\ -\ 1x01\ -\ Pilot\ -\ 720p\ BluRay.mkv
ffprobe version 3.4.4 Copyright (c) 2007-2018 the FFmpeg developers
  built with gcc 6.4.0 (Alpine 6.4.0)
  configuration: --prefix=/usr --enable-avresample --enable-avfilter --enable-gnutls --enable-gpl --enable-libass --enable-libmp3lame --enable-librtmp --enable-libvorbis --enable-libvpx --enable-libxvid --enable-libx264 --enable-libx265 --enable-libtheora --enable-libv4l2 --enable-postproc --enable-pic --enable-pthreads --enable-shared --enable-libxcb --disable-stripping --disable-static --enable-vaapi --enable-vdpau --enable-libopus --disable-debug
  libavutil      55. 78.100 / 55. 78.100
  libavcodec     57.107.100 / 57.107.100
  libavformat    57. 83.100 / 57. 83.100
  libavdevice    57. 10.100 / 57. 10.100
  libavfilter     6.107.100 /  6.107.100
  libavresample   3.  7.  0 /  3.  7.  0
  libswscale      4.  8.100 /  4.  8.100
  libswresample   2.  9.100 /  2.  9.100
  libpostproc    54.  7.100 / 54.  7.100
Input #0, matroska,webm, from '/library/Season 01/Breaking Bad - 1x01 - Pilot - 720p BluRay.mkv':
  Metadata:
    ENCODER         : Lavf57.83.100
  Duration: 00:58:06.54, start: 0.000000, bitrate: 2137 kb/s
    Chapter #0:0: start 0.000000, end 247.080000
    Metadata:
      title           : 01
    Chapter #0:1: start 247.080000, end 967.216000
    Metadata:
      title           : 02
    Chapter #0:2: start 967.216000, end 1901.149000
    Metadata:
      title           : 03
    Chapter #0:3: start 1901.149000, end 2910.658000
    Metadata:
      title           : 04
    Chapter #0:4: start 2910.658000, end 3432.262000
    Metadata:
      title           : 05
    Chapter #0:5: start 3432.262000, end 3486.518000
    Metadata:
      title           : 06
    Stream #0:0(eng): Video: hevc (Main), yuv420p(tv, progressive), 1280x720 [SAR 1:1 DAR 16:9], 23.98 fps, 23.98 tbr, 1k tbn, 23.98 tbc
    Metadata:
      ENCODER         : Lavc57.107.100 libx265
      DURATION        : 00:58:06.510000000
    Stream #0:1(eng): Audio: dts (DTS), 48000 Hz, 5.1(side), fltp, 1536 kb/s (default)
    Metadata:
      DURATION        : 00:58:06.535000000
    Stream #0:2(eng): Audio: aac (LC), 48000 Hz, stereo, fltp (default)
    Metadata:
      title           : 'Stereo'
      ENCODER         : Lavc57.107.100 aac
      DURATION        : 00:58:06.536000000
```
All files in the pending tasks were already completed by unmanic.

**Comment by PutzJimmy** [comment]
I can confirm this happens for me too

**Comment by marcosviniciusi** [comment]
I had the same Problem

**Comment by Josh5** [comment]
The changes being written with https://github.com/Josh5/unmanic/projects/3 completely change how this work. All going to plan this should be merged in then next couple of days.
Could I please ask you to test again once it has been merged. From there if the issue still persists, I will look further into it

**Comment by Josh5** [comment]
Fixed in commit e0a5c8821756bcb70f3a1e21b5b8b137e312cc59
